docs/integrations/microsoft-azure/azure-container-instances.md

```diff
@@ -0,0 +1,210 @@
+   ```
+
+### Configure metric rules
+
+Create the following metrics rules by following the instructions in [Create a metrics rule](/docs/metrics/metric-rules-editor/#create-a-metrics-rule).
+
+#### Azure observability metadata extraction service level
+
+If this rule already exists, there is no need to create it again.
+
+   ```sql
+   Rule Name: AzureObservabilityMetadataExtractionServiceLevel    
+   ```
+   
+   ```sql title="Metric match expression"
+   resourceId=/SUBSCRIPTIONS/*/RESOURCEGROUPS/*/PROVIDERS/*/*/*/*/* tenant_name=*
+   ```
+   | Fields extracted | Metric rule    |
+   |:-----------------|:---------------|
+   | subscription_id  | $resourceId._1 |
+   | resource_group   | $resourceId._2 |
+   | provider_name    | $resourceId._3 |
+   | resource_type    | $resourceId._4 |
+   | resource_name    | $resourceId._5 |
+   | service_type     | $resourceId._6 |
+   | service_name     | $resourceId._7 |
+
+#### Azure observability metadata extraction container instance level
+
+   ```sql
+   Rule Name: AzureObservabilityMetadataExtractionAzureContainerInstanceLevel
+   ```
+
+   ```sql title="Metric match expression"
+   resourceId=resourceId=/SUBSCRIPTIONS/*/RESOURCEGROUPS/*/PROVIDERS/MICROSOFT.CONTAINERINSTANCE/*/* tenant_name=*
+   ```
+   | Fields extracted  | Metric rule                 |
+   |:------------------|:----------------------------|
+   | subscription_id   | $resourceId._1              |
+   | resource_group    | $resourceId._2              |
+   | provider_name     | MICROSOFT.CONTAINERINSTANCE |
+   | resource_type     | $resourceId._3              |
+   | resource_name     | $resourceId._4              |
+
+### Configure metrics collection
+ 
+ For metrics collection follow guidelines in [Azue Metrics Source](/docs/send-data/hosted-collectors/microsoft-source/azure-metrics-source/).
+ 
+ While you configure metrics collection you need to tag the location field in the source with right location value. <br/><img src={useBaseUrl('img/integrations/microsoft-azure/Azure-Storage-Tag-Location.png')} alt="Azure Container Instance Tag Location" style={{border: '1px solid gray'}} width="400" />
+ 
+ Also you need to configure namespaces as shown below. <br/><img src={useBaseUrl('img/integrations/microsoft-azure/azure-container-instance-namespaces.png')} alt="Azure Container Instance Namespaces" style={{border: '1px solid gray'}} width="500" />
+
+### Configure logs collection
+1. Add a hosted collector and [HTTP Source](/docs/send-data/collect-from-other-data-sources/azure-monitoring/collect-metrics-azure-monitor/#step-1-configure-an-http-source).
+2. Create and push a custom image using a <a href="/files/Dockerfile" target="_blank">Docker file</a> and <a href="/files/output_conf.yaml" target="_blank">output_conf.yaml</a> onto a Docker hub.
+3. Create new resource group in Azure.
+4. Deploy the <a href="/files/logging-sidecar-deploy.yaml" target="_blank">logging-sidecar-deploy.yaml</a> Azure template.
```

Comment:
Let's add azure cli commands also for deploying container group and creating resource group